automl/HPOlib3
# Development

- Container for cartpole, xgb, learna, nasbench101, nasbench201, tabular_benchmarks
- Test cases
- More examples. Existing examples work now with container
- Interface update: All interfaces are more or less now equal. Take a rng as input.
- Update of the container interface
- Updtae of the server logic
- Update of the requirements. We need to restrict pandas to <=0.24.
- Update aof recipes
- Readme contains now an overview of included benchmarks.

NOTE: Travis will fail, because I'm currently uploading the benchmarks to sylabs. 
After they are uploaded and travis is rerun, they should work.
